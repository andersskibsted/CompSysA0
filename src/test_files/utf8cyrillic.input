	в	г	д	—	е	ж	з	и	й	—	к	л	—	м	н	—	о	п	р	с	