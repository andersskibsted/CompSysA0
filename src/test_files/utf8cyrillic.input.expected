test_files/utf8cyrillic.input: Unicode text, UTF-8 text, with no line terminators
